Makefile

```diff
@@ -1,6 +1,7 @@
 DESTDIR :=
-BINDIR := /usr/local/bin
-SYSCONFDIR := /etc
+PREFIX := /usr/local
+BINDIR := $(PREFIX)/bin
```

Comment:
I also prefer to have the default prefix as /usr rather than /usr/local, as we pretty much use that for all our other binaries.